citi-onboarding/Aqua-Jr
# Feature/area footer data base route

# What i did: 
- Create the Areas table in the data base and create a route.
- Create the  table in the data base and create a route.
# How to test:
1. in your terminal run ` git checkout develop`
2. in your terminal run ` git pull `
3. in your terminal run `git checkout remotes/origin/feature/Area-Footer-DataBase-Route`
4. in your terminal run `cd server`
5. in yout terminal run `node index.js` or `nodemon index.js`
6. access `localhost:3001/admin`
7. create a object of 'Areas' and access `localhost:3001/api/areas`
8. edit the object of 'Footers' and access `localhost:3001/api/footers`
9. if the object you created appears in int the end point 'api/[model]' it´s works.
